webcompat/form.py

```diff
@@ -129,7 +130,7 @@ def get_details(details):
         rv = ''.join(['<li>{k}: {v}</li>'.format(k=k, v=get_str_value(v))
                       for k, v in details.items()])
     except AttributeError:
-        return content
+        return '<li>{content}</li>'.format(content=content)
```

Comment:
This seems like such an obviously good idea... wish I would have thought of it. :)

Reply:
❤️